src/nrniv/CMakeLists.txt

```diff
@@ -386,6 +393,11 @@ if(NOT NRN_MACOS_BUILD AND READLINE_FOUND)
   target_link_libraries(nrniv ${Readline_LIBRARY})
 endif()
 
+if(NRN_ENABLE_MEMACS)
+  add_executable(memacs ${PROJECT_SOURCE_DIR}/src/memacs/main.c ${NRN_MEMACS_SRC_FILES})
```

Comment:
curious - do we still need micro emacs?

Reply:
I do. Desperately.  Otherwise, editing for me like swimming in molasses.

Reply:
Note that memacs is off by default and not distributed for any binary.

Reply:
> I do. Desperately. Otherwise, editing for me like swimming in molasses.

Ok 😃. But if this is not needed for end users, I assume this could be moved out of neuron source tree in the future. (I think @alexsavulescu was saying something similar)

Reply:
I thought the userbase was ~0 :)

Reply:
we can ship separate python wheel : pip install memacs 😄

Reply:
Already there: https://pypi.org/project/memacs/ 
😄

Reply:
Yeah, it was just a joke :D